tests/test_event_listener.rs

```diff
@@ -109,6 +121,13 @@ fn test_event_listener_basic() {
     assert_ne!(counter.compaction.load(Ordering::SeqCst), 0);
     drop(db);
     assert_eq!(counter.drop_count.load(Ordering::SeqCst), 1);
+    assert!(
+        counter.input_records.load(Ordering::SeqCst) >
```

Comment:
why input is bigger than output here?

Reply:
Because we have written duplicated keys.

Reply:
Got it.